src/main/kotlin/no/nav/cv/eures/konverterer/CvAvroSchema.kt

```diff
@@ -1,14 +1,15 @@
 package no.nav.cv.eures.konverterer
```

Comment:
Tror ikke vi bruker denne lenger, etter at vi la til avhengigheten med Avro skjema for CV?

Reply:
Tar vi bort den i en annen PR kanskje? Etter vi har fått det til å kjøre i test